app/views/tasks/index.html.erb

```diff
@@ -8,7 +8,9 @@
     <tr>
```

Comment:
We should probably drop all this. Tasks don't have a lot of value outside the context of an event.

After Monday 😄